content/geometry/Point.h

```diff
@@ -9,6 +9,7 @@
  */
 #pragma once
 
+template <typename T> int sgn(T x) { return (x > 0) - (x < 0);}
```

Comment:
`template<class T>` for consistency (and it's shorter so you can add back the space at the end)